src/components/Message/Message.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { Img, Text } from '../styles/UI';
+import { useRecoilState } from 'recoil';
+import { UserState } from '../../atoms';
+
+export default function Message({ message, isOwner }) {
+  const [users, setUser] = useRecoilState(UserState);
+
+  const user = users.find((u) => u.userId === message.userId);
+
+  const formatTimestamp = (timestamp) => {
```

Comment:
컴포넌트의 props나 함수의 인자값은 꼭 타입을 지정해 주는 게 좋습니다!
그리고 `timestamp`라는 이름은 보통 `1728916619` 이런 형식의 Unix timestamp를 사용할 때에만 쓰는 것 같아요 😶
```suggestion
  const formatTimestamp = (time: string) => {
```